Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```diff
@@ -291,6 +312,16 @@ script:
       name: ip_rules
       required: false
       secret: false
+    - name: subscription_id
+      description: "The subscription ID. Note: This argument will override the instance parameter ‘Subscription ID'."
```

Comment:
Now we should change the name here to the new display name of ‘Subscription ID'. 
Relevant in all commands, and also relevant for resource group.

Reply:
Fixed.